src/KafkaFlow/Consumers/Consumer.cs

```diff
@@ -164,25 +164,27 @@ private void EnsureConsumer()
             this.consumer =
                 consumerBuilder
                     .SetPartitionsAssignedHandler(
-                        (consumer, partitions) =>
+                        (c, partitions) =>
```

Comment:
I prefer `consumer `instead of `c`

Reply:
`consumer parameter` was hiding the `consumer property`, but it can still be accessed using `this`... renamed to `consumer `again,